plugins/chat/assets/javascripts/discourse/components/chat/message-creator.js

```diff
@@ -453,6 +453,10 @@ export default class ChatMessageCreator extends Component {
     this.chat
       .upsertDmChannelForUsernames(users.mapBy("username"))
       .then((channel) => {
+        if (!channel) {
+          return;
```

Comment:
Sorry missed the question, yes that's exactly why, I hope we can in the near future get away with this `channel key=usernames` pattern, it's historic choices. Now that we will move to private groups, we can now have proper channels which are directed by an abstract entity (`ChannelGroup` or something), just like we have `Category` for category channels, it will avoid all kind of stuff like this...